src/waveform/renderers/allshader/waveformrendererslipmode.cpp

```diff
@@ -44,6 +44,11 @@ void WaveformRendererSlipMode::draw(QPainter* painter, QPaintEvent* event) {
 bool WaveformRendererSlipMode::init() {
     m_timer.restart();
 
+    if (m_waveformRenderer->getGroup().isEmpty()) {
+        m_pSlipModeControl.reset();
+        return true;
```

Comment:
this seems like an error case, shouldn't it return false?

Reply:
It's not - this is the case where we are rendering a scrolling waveform without a deck attached to it (using explicit playback position). See justification above